metricflow/model/semantics/linkable_spec_resolver.py

```diff
@@ -838,12 +860,14 @@ def _get_linkable_element_set_for_measure(
         measure_semantic_model = self._get_semantic_model_for_measure(measure_reference)
 
         elements_in_semantic_model = self._get_elements_in_semantic_model(measure_semantic_model)
+        metrics_linked_to_semantic_model = self._get_joinable_metrics_for_semantic_model(measure_semantic_model)
         metric_time_elements = self._get_metric_time_elements(measure_reference)
         joined_elements = self._get_joined_elements(measure_semantic_model)
 
         return LinkableElementSet.merge_by_path_key(
             (
                 elements_in_semantic_model,
+                metrics_linked_to_semantic_model,
```

Comment:
Oh, so this because all of our other joinable metrics were pathing through an entity, so we were finding targets through the existing mechanisms, but since the metrics are a new linkable type they weren't getting included if there was no need for an additional join through an entity or whatever. Is that correct?

Reply:
Yes correct!
This is basically adding "local linkable metrics" to the list of joinable metrics. Technically they aren't local since they aren't necessarily defined in the semantic model. But they are joined once via an entity local to the semantic model.